content/blog/2024-01-10-meeting.md

```diff
@@ -34,7 +34,7 @@ Here is a summary of the outcomes from new year consultations:
 
 - Allocation has been made to fund several bounties, which will be documented clearly.
 - There was a review of existing services and opening up the opportunity to hire some staff for web site and repository maintenance.  Further details to be irons out with community feedback.
-- The first bounty has [been drafted](https://github.com/orgs/MonoGame/projects/1?pane=info), with the aim for the MonoGame board to deliver an initial specification implementation and then open up for community feedback and contributions.  Of note, the team feels it is best to deliver the framework that the community can build upon.
+- The first bounty has been drafted for NativeAOT (however, it is under NDA and only registered members can access it), with the aim for the MonoGame board to deliver an initial specification implementation and then open up for community feedback and contributions.  Of note, the team feels it is best to deliver the framework that the community can build upon.
```

Comment:
@SimonDarksideJ just change this to say it will be published soon.

The issue was that we don't intend the project system to post them.

That is the plan i believe... correct @mrhelmut   ?